src/menu/common.c

```diff
@@ -6,15 +6,16 @@
  * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
  */
 
+#include "progress.h"
 #include "taisei.h"
```

Comment:
Never include anything before `taisei.h`, it may confuse the `ninja upkeep` script.